Chargebee/Classes/Purchase/CBPurchaseManager.swift

```diff
@@ -12,7 +12,7 @@ public class CBPurchase: NSObject {
     public static let shared = CBPurchase()
     private var productIDs: [String] = []
     public var receiveProductsHandler: ((_ result: Result<[CBProduct], CBPurchaseError>) -> Void)?
-    public var buyProductHandler: ((Result<(status:Bool, subscriptionId:String?, planId:String?), Error>) -> Void)?
+    public var buyProductHandler: ((Result<(status: Bool, subscriptionId: String?, planId: String?, customerId: String?), Error>) -> Void)?
```

Comment:
This would be a breaking change. Is there anything we can do to reduce the impact?
Also would prefer this tuple be a type/alias instead.

Reply:
> Would this be a breaking change. Is there anything we can do to reduce the impact? Also would prefer this tuple be a type/alias instead.

Updated with struct